app/Providers/ModelServiceProvider.php

```diff
@@ -45,6 +45,8 @@ public function boot()
                 in_array($post->text, [
                     'Test runscope upload',
                     'caption_ghost_test',
+                ]) || in_array($post->why_participated, [
```

Comment:
`why_participated` lives on the `signup` - is there a workaround that lets us grab this directly from the post?

Reply:
Ohhhhh dip! I suppose we can just grab it from the associated signup right? 
```suggestion
                ]) || in_array($post->signup->why_participated, [
```

Reply:
Yeah I think that should work! We [do that elsewhere](https://github.com/DoSomething/northstar/blob/4dbda054e21cffcce1e43cd882f2f647a2356607/app/Notifications/PostTagged.php#L90)

Reply:
Magnifique! I'll commit this suggestion 🏗️